src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidatorTest.java

```diff
@@ -22,34 +27,115 @@ class RemoveValidatorTest {
+    }
+
+    @Test
+    void testFullValidationErrorReportedWhenFilingDateIsPresentAndIsNotProprietorOfLandIsTrue() {
+        removeDto.setIsNotProprietorOfLand(true);
+        updateDto.setFilingDate(LocalDate.of(2024, 2, 1));
+
+        Errors errors = removeValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFieldName = UPDATE_FIELD + "." + UpdateDto.FILING_DATE;
+        String validationMessage = ValidationMessages.SHOULD_NOT_BE_POPULATED_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testFullValidationErrorsReportedWhenFilingDateIsPresentAndIsNotProprietorOfLandIsFalse() {
+        removeDto.setIsNotProprietorOfLand(false);
+        updateDto.setFilingDate(LocalDate.of(2024, 2, 1));
+
+        Errors errors = removeValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFilingDateFieldName = UPDATE_FIELD + "." + UpdateDto.FILING_DATE;
+        String filingDateValidationMessage = ValidationMessages.SHOULD_NOT_BE_POPULATED_ERROR_MESSAGE.replace("%s", qualifiedFilingDateFieldName);
+        String qualifiedRemoveStatementFieldName = getQualifiedFieldName(RemoveDto.IS_NOT_PROPRIETOR_OF_LAND_FIELD);
+        String removeStatementValidationMessage = ValidationMessages.NOT_VALID_ERROR_MESSAGE.replace("%s", qualifiedRemoveStatementFieldName);
+        assertError(qualifiedFilingDateFieldName, filingDateValidationMessage, errors);
+        assertError(qualifiedRemoveStatementFieldName, removeStatementValidationMessage, errors);
+    }
+
+    @Test
+    void testFullValidationErrorReportedWhenFilingDateIsNotPresentAndRemoveBlockNotPresent() {
+        overseasEntitySubmissionDto.setRemove(null);
+
+        Errors errors = removeValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFieldName = getQualifiedFieldName(RemoveDto.IS_NOT_PROPRIETOR_OF_LAND_FIELD);
+        String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testFullValidationErrorReportedWhenFilingDateIsNotPresentAndIsNotProprietorOfLandIsNull() {
+        overseasEntitySubmissionDto.getRemove().setIsNotProprietorOfLand(null);
+
+        Errors errors = removeValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFieldName = getQualifiedFieldName(RemoveDto.IS_NOT_PROPRIETOR_OF_LAND_FIELD);
+        String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testFullValidationErrorReportedWhenUpdateBlockNotPresentAndIsNotProprietorOfLandIsTrue() {
```

Comment:
Does the UpdateDto need be populated for a Remove journey even if it's a no change journey?